Worker.Extension.Sql/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.Sql", "0.1.457-preview")]
```

Comment:
I would prefer that we use the same version number as the build for this instead of hardcoding it - that way we save ourselves having to update this every time we want to release (and when building/running from source we should be using the local dev copy of the extension anyways - otherwise we end up with situations like the python tests where we're only testing with released code)

So the version here would be "99.99.99" and then during the pipeline builds we'd replace that value with the actual nuget package version we generated.

Reply:
I chatted with @fabiocav regarding have it updated during build and he said "You want to, at the very least, lock to a given major version. You can use "x.*" (e.g., 3.*), which would get the highest 3.x release", Am checking with him if we can do 0.1.*-preview

Reply:
You can just try it out yourself, right? It sounds like there isn't an issue with using version ranges and I definitely agree locking it to a major version (+ -preview) is what we'd want to do right now. `0.*-preview` should be good if that's allowed.